website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -48,7 +48,6 @@ const sx: SystemStyleObject = {
   },
   p: {
     whiteSpace: "pre-wrap",
-    display: "inline-block",
```

Comment:
fixed #1657. I added inline-block because I want to keep the old behavior that text will be inlined with the avatar on mobile but that caused so many problems so I decide to drop it.